internal/provisioner/registry/finalizer/deletecachedbundle.go

```diff
@@ -0,0 +1,22 @@
+package finalizer
```

Comment:
Any idea on why we couldn't have this live as a top-level internal package that both provisioner implementations can use? I don't see any noticable distinction between the registry + plain finalizer logic unless I'm missing something obvious.

Reply:
Resolved in #390